src/status_im/events.cljs

```diff
@@ -2,7 +2,7 @@
   (:require
    clojure.set
    [re-frame.core :as re-frame]
-   status-im.add-new.core
+   status-im2.contexts.add-new-contact.events
```

Comment:
please move it to `status-im2.setup.events`

Reply:
I'm couldn't figure out what you meant here, but I deleted the `:require` from `src/status_im/events.cljs` (doesn't seem to be needed)

Reply:
you still need to require it once somewhere, so re-frame events are registered , i just suggerested to move `status-im2.contexts.add-new-contact.events` to `status-im2.setup.events` namespace

Reply:
oh yeah, got it! thanks!